R/output.R

```diff
@@ -699,6 +699,9 @@ knit_print.default = function(x, ..., inline = FALSE) {
 #' @export
 knit_print.knit_asis = function(x, ...) x
```

Comment:
Can roxygen2 recognize two function names if we write the code this way?

```suggestion
knit_print.knit_asis = knit_print.knit_asis_url = function(x, ...) x
```

If not, adding a new function on a separate line is fine to me.

Reply:
It seems like roxygen2 does not support it. The S3 method is not exported in NAMESPACE in this case.